File: model/Tag.js
import mongoose from 'mongoose';

const tagSchema = new mongoose.Schema({
    name: {
        type: String,
        minlength: 2, 
        maxlength: 50
    },
    dateTime: {
        type: Number
    },
    moodValue: {
        type: Number
    },
    recent: {
        type: Number
    }
}); 

const Tag = mongoose.model("Tag", tagSchema, "Tag"); 
export { Tag }

File: FrontendSwift/MoodTracker/Model/Tag.swift
//
//  Tag.swift
//  MoodTracker
//
//  Created by Ivy Tan on 8/3/22.
//

import Foundation

struct Tag : Codable, Hashable {
//    var _id : String?
    var name: String
//    var dateTime: Date
//    var moodValue: Float
    var recent: Int
}
